dataPipelines/gc_amhs_pipeline/gc_pipeline.py

```diff
@@ -0,0 +1,67 @@
+from time import time
+import os
+from typing import Union
+from pathlib import Path
+import click
+import csv
+from elasticsearch import helpers, Elasticsearch
+import json
+from .conf import Conf
+from dataPipelines.gc_amhs_pipeline import amhs_parser as parser
```

Comment:
two things here:
1. we'll need to move the parser (amhs_parser.py) into common/document_parser/parsers
2. then just change this import statement to import from that new location